### ISSUE TITLE
R15rc1 bugs: affiliation

### ISSUE THREAD

**Issue by jimmyzhen** [ClinGen/clincoded#1570]
@jimmyzhen When testing together, we found a bug. It's a very specific chain of events that generates the error.

Selina and I were both logged in as part of the same #100003 Affiliation (see https://r15-rc1.demo.clinicalgenome.org/curation-central/?gdm=8b5732d4-cfe7-4941-8940-04650a56a832&pmid=10000).
When both logged in everything worked fine, except some Probands became uneditable to us despite the fact that they were added by me as part of #100003:

<img width="189" alt="screen shot 2017-12-21 at 8 20 51 am" src="https://user-images.githubusercontent.com/15131169/34264477-e9c3c6c6-e627-11e7-816a-655c58f376d6.png">

We should both have been able to edit this because we were both #100003.

We found out that the only effected Probands were ones that were created as part of a Group and Family. If the proband was created separately then there were no issues.

A second surprising thing, is that only I could generate the error. If Selina created a Proband from a Group and Family, it worked just fine. We found out that it was because I was the creator of the PMID because when Selina added her own PMID, then she was able to recreate the problem:

<img width="259" alt="screen shot 2017-12-21 at 8 44 00 am" src="https://user-images.githubusercontent.com/15131169/34265594-2e1c8b7a-e62b-11e7-86f4-a1845c2229e8.png">

So, in short:
If a curator (associated with an Affiliation) adds a PMID and then they generate a Proband as part of a Group and Family, then that Proband becomes uneditable.

One more thing.... I just logged out of the Affiliation and logged in as myself (No Affiliation) and that Proband (that was created as part of #10003) is now now editable:
<img width="261" alt="screen shot 2017-12-21 at 8 31 31 am" src="https://user-images.githubusercontent.com/15131169/34265062-b6d87bf6-e629-11e7-8f0c-4627095831a8.png">

**Comment by selinad** [comment]
HI @jimmyzhen wrt to change on the Dashboard for Affiliations.

Keep the GDM title, but add asterisk.  Then, below GDM table, include this text (should it go in blue box?):

"* Gene-Disease records created by this Affiliation.  To create a new Gene-Disease record, click on "New Gene Curation" in the header; to find an existing record not created by this Affiliation, click on "View All Gene:Disease Records" under "Tools" and use the filter box to search by HGNC gene symbol"

@wrightmw please feel free to edit.

**Comment by selinad** [comment]
The "Preview" watermark makes PMIDs etc. unclickable, so @jimmyzhen will move to "back" instead of having it sit on top.

**Comment by jimmyzhen** [comment]
@selinad @wrightmw,

How does the "note" look?

![image](https://user-images.githubusercontent.com/6956310/34273596-e57888e6-e649-11e7-8827-8a40e45205a9.png)

**Comment by wrightmw** [comment]
@jimmyzhen Looks good! But bottom of table would look better :-)

**Comment by selinad** [comment]
Uh oh -- I clicked to add Disease and got this (you can see the blue Disease+ button):

![image](https://user-images.githubusercontent.com/11320314/34275411-0db721d4-e652-11e7-87e3-b4e526bec849.png)

**Comment by wrightmw** [comment]
Ooer!

**Comment by selinad** [comment]
@wrightmw I was fooled by button color -- behavior is right, but button should be red (it is in demo).  @jimmyzhen looking into it.

**Comment by jimmyzhen** [comment]
The color of the Delete Disease button looks okay too....
![image](https://user-images.githubusercontent.com/6956310/34275667-2e8d9fea-e653-11e7-8b7e-4490d53e11eb.png)

**Comment by selinad** [comment]
@jimmyzhen I'm confused as it didn't look ok for me (as you saw).

**Comment by jimmyzhen** [comment]
Try a different browser and see if you can reproduce the problem.

**Comment by selinad** [comment]
I think it happens when I go back to Interpretation from the Summary....just saw it red in the interpretation matt sent and then after I went to summary and came back it was blue

**Comment by selinad** [comment]
Why are there 2 interpretations for same variant as part of DCM?   They have different evaluations -- let's start a new one, @wrightmw 

![image](https://user-images.githubusercontent.com/11320314/34275915-1d2c5894-e654-11e7-9f92-4c301f873d6d.png)

**Comment by selinad** [comment]
I know for a fact I chose PVS1 and now it is gone:
![image](https://user-images.githubusercontent.com/11320314/34276128-e2919496-e654-11e7-8dc6-4cd7ce4f83d3.png)

**Comment by jimmyzhen** [comment]
Included in R15 release.
